test/test_images/hellovolume/service.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: hello-volume
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            # This is the Go import path for the binary to containerize
+            # and substitute here.
+            image: github.com/knative/serving/test/test_images/hellovolume
+            volumeMounts:
+            - name: foo
+              mountPath: /hello
+          volumes:
+          - name: foo
+            secret:
+              secretName: bar
```

Comment:
Add a comment that this will be replaced at test execution time?

Reply:
This yaml isn't actually used for the test, but enables folks to stand up a Service in the same manner as the test.  This is something @dgerd has been doing for a bunch of the e2e tests.